protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -774,8 +773,12 @@ func applyUpdatesToPositions[
 		id := update.GetId()
 		_, exists := updateMap[id]
 		if exists {
-			errMsg := fmt.Sprintf("Multiple updates exist for position %v", update.GetId())
-			return nil, errorsmod.Wrap(types.ErrNonUniqueUpdatesPosition, errMsg)
+			panic(
+				fmt.Sprintf(
+					"applyUpdatesToPositions: Multiple updates exist for position %v",
+					update.GetId(),
+				),
+			)
```

Comment:
Would anyone prefer I make these error messages use `log.NewLazySprintf`? This avoids unnecessarily string concatenation in the hot path.

If anyone wants me to do that, I would prefer to make a ticket to do it later. I can do it now though if anyone would prefer.